python/mxnet/contrib/onnx/mx2onnx/export_onnx.py

```diff
@@ -246,21 +251,26 @@ def create_onnx_graph_proto(self, sym, params, in_shape, in_type, verbose=False,
                     mx_graph=mx_graph,
                     weights=weights,
                     in_shape=in_shape[graph_input_idx],
-                    in_type=in_type,
+                    in_type=in_type[graph_input_idx],
                     proc_nodes=all_processed_nodes,
                     initializer=initializer,
                     outputs_lookup=outputs_lookup)
                 graph_input_idx += 1
 
             else:
+                # Handle no input case
```

Comment:
I think this intype is used mostly in operators like `plus_scalar`, where we need to cast the parameter `scalar` to the same type as `data`. So far the models we export do not have any issue with this, but a possible error is that the dtype of `data` is different from intype. If we ever run into such a problem we can consider requiring a dict from user, which specifies the intype's for specific operators

Reply:
This would benefit all operators with multiple input types. For instance `where`, before this PR data type of `x` and `y` is masked by `condition`, means the output will always be converted to `int32` (it passes the unittest with `float64` although `where` does not support `float64`). After this PR the output will have the correct data type.